syntax ="proto3";

package lrlb;

// lrlb 系统的消息ID

enum MessageID{
	ID_UNKNOW  =0; //proto3 enum第一个属性必须为0
	ID_GetRouteRequest  =1;  //向DNS请求Route对应的关系的消息ID
	ID_GetRouteResponse =2;  //DNS回复Route信息的消息ID
}

//一个管理主机信息

message HostInfo{
	int32 ip=1;
	int32 port=2;
}

//请求lrlb-dns route请求信息的消息内容
message GetRouteRequest{
	int32 modid=1;
	int32 cmdid=2;
}

//lrlb-dns 回复route信息消息内容
message GetRouteResponse{
	int32 modid=1;
	int32 cmdid=2;
	repeated HostInfo host=3;
}





